src/main/java/lesson03/conditions/BmiInfo.java

```diff
@@ -0,0 +1,51 @@
+package lesson03.conditions;
+/*
+Вычислить BMI (индекс массы тела) человека по показателям веса (в килограммах) и роста (в метрах).
+В результате вернуть текстовое описание BMI.
+bmi = weight / height ^ 2, где ^ - возведение в степень.
+
+Текстовое описание:
+bmi <= 18.5                -> "Underweight"
+18.5 < bmi <= 25.0     -> "Normal"
+25.0 < bmi                  -> "Overweight"
+
+Использовать конструкцию if-else-if.
+Ограничения: weight > 0, height > 0.
+String bmiInfo(double weight, double height)
+Входные аргументы double weight, double height
+Тип результата String
+*/
+
+import utils.errorhandling.ExceptionHandling;
+
+public class BmiInfo {
+    public static final String BMI_UNDERWEIGHT = "Underweight";
+    public static final String BMI_NORMAL = "Normal";
+    public static final String BMI_OVERWEIGHT = "Overweight";
+
+    public static void main (String[] args) {
+        System.out.println(bmiInfo(-175, -1.80));
+    }
+
+    public static String bmiInfo(double weight, double height){
+        String bmiResultDescription = "";
+        if (isBioParametersCorrect(weight, height)){
+            double bmi = weight / Math.pow(height, 2);
+            if (bmi <= 18.5){
+                bmiResultDescription = BMI_NORMAL;
+            } else {
```

Comment:
Try reducing complexity of method:
if (condition) {
} else if (condition) {
} else {
}

Reply:
I removed first if that was reponsible for input validation. However if-else-if usage was the part of the task.